remove-data/src/main/java/eu/europeana/metis/remove/plugin/RemovePluginsMain.java

```diff
@@ -0,0 +1,240 @@
+package eu.europeana.metis.remove.plugin;
+
+import com.opencsv.CSVReader;
+import eu.europeana.metis.core.dao.WorkflowExecutionDao;
+import eu.europeana.metis.core.mongo.MorphiaDatastoreProvider;
+import eu.europeana.metis.core.workflow.WorkflowExecution;
+import eu.europeana.metis.core.workflow.WorkflowStatus;
+import eu.europeana.metis.core.workflow.plugins.AbstractMetisPlugin;
+import eu.europeana.metis.core.workflow.plugins.PluginStatus;
+import eu.europeana.metis.core.workflow.plugins.PluginType;
+import eu.europeana.metis.remove.utils.Application;
+import eu.europeana.metis.utils.ExternalRequestUtil;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.EnumSet;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.IntStream;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.lang3.tuple.ImmutablePair;
+import org.apache.commons.lang3.tuple.Pair;
+import org.apache.logging.log4j.core.net.ssl.TrustStoreConfigurationException;
+import org.mongodb.morphia.query.FindOptions;
+import org.mongodb.morphia.query.Query;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class RemovePluginsMain {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(RemovePluginsMain.class);
+
+  private static final String FILE_FOR_PLUGIN_REMOVAL = "/home/jochen/Desktop/plugins_to_remove.csv";
+
+  public static void main(String[] args) throws TrustStoreConfigurationException, IOException {
+    try (final Application application = Application.initialize()) {
+      final List<PluginToRemove> plugins = readFile();
+      LOGGER.info("Removing {} plugins.", plugins.size());
+      final WorkflowExecutionDao dao = new WorkflowExecutionDao(application.getDatastoreProvider());
+      plugins.stream().map(plugin -> RemovePluginsMain.findPlugin(plugin, dao))
+          .filter(Objects::nonNull).filter(plugin -> canEditWorkflow(plugin.getLeft()))
```

Comment:
The parameter "plugin" in the lambdas is a pair of execution and plugin, perhaps rename it?

Reply:
Fixed.